### PR TITLE
documentation: updating SageMaker ReadTheDocs

### PR DESCRIPTION
**Pull request by chrstfu**
### DO NOT MERGE: This documentation changes still need updates to the new notebook URLs

*Description of changes:*
Some updates for [the public documentation](https://sagemaker.readthedocs.io/en/stable/algorithms/index.html). The updates were done according to [this document](https://quip-amazon.com/lW3iAcmNIaxe/UpdatesFixes-in-httpssagemakerreadthedocsioenstablealgorithmsindexhtml).

*Testing done:*
Ran `make clean html` and manually verified the asks in the document above were met.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: doc/doc_utils/jumpstart_doc_utils.py [hunk 1]
```diff
@@ -149,8 +155,48 @@ class Frameworks(str, Enum):
...
+    (Tasks.SPC): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/Amazon_JumpStart_Sentence_Pair_Classification.ipynb",
+    (Tasks.TC): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb",
+    (Tasks.TC_EMBEDDING): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_text_embedding/Amazon_JumpStart_Text_Embedding.ipynb",
+    (Tasks.TEXT_GENERATION): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_text_generation/Amazon_JumpStart_Text_Generation.ipynb",
+    (Tasks.SUMMARIZATION): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_text_summarization/Amazon_JumpStart_Text_Summarization.ipynb",
+    # TODO: Find notebook link for  this task
+    (Tasks.ZSTC): "https://www.google.com/",
```
Comment (JGuinegagne) [inline doc/doc_utils/jumpstart_doc_utils.py:192]: maybe we should use `amazon.com` as placeholder ;)

Comment (chrstfu) [reply doc/doc_utils/jumpstart_doc_utils.py:192]: How does rickrolling our customers sound? I can put in that url lol

**Comment by mufaddal-rohawala** [comment]
@chrstfu is this still relevant?

**Comment by mufaddal-rohawala** [comment]
Closing this, @chrstfu please reopen if this is still relevant.

**Pull request closed without merging**